llvm/lib/Target/AIE/AIE2MultiSlotPseudoInstrInfo.td

```diff
@@ -12,6 +12,8 @@
 // The last parameter to the MulitSlot_Pseudo is instruction into which the Multi-Slot Pseudo could be materialize to.
 // Priority to Slot/Instruction is based on the sequence in which the real instructions are passed
 
+// Note : Any of the Itinerary could be selected, but you need to make sure the operand latency is same for all the possible instr.
```

Comment:
I think it's the other way. We can only use a multislot pseudo for instructions that have equal latencies.

Reply:
Correct me if I am wrong my interpretation of `same operand latency` is same as `equal latencies`

Reply:
I'm not denying that. I would just state that the itineraries of all instructions covered by a multi-slot pseudo should be equivalent. It's not part of selecting the itinerary, it's part of defining the MSPseudo.